### DISCUSSION TITLE
livewire.min.js  is 404. What's a hell? Please, help me.

### DISCUSSION THREAD

**Discussion by cleargoal** [filamentphp/filament#12641] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

The browser responds with an error 

> livewire.min.js **not found**

on the production server, while at local it's OK, but uses 'livewire.js' - not minified.
As a result, no panel page was rendered.

How to find and fix the problem?

**Please, any help is appreciated.**

**Comment by leandrocfe** [comment]
livewire.min? Where is it?

**Reply by cleargoal** [reply]
Actually, it is on the path
vendor/livewire/livewire/dist/livewire.min.js

And there are 3 calls to it from the file
vendor/livewire/livewire/src/Mechanisms/FrontendAssets/FrontendAssets.php

**Reply by cleargoal** [reply]
Solved.
I forgot to publish livewire assets, between which is 'livewire.min.js' and this circumstance was born an error.
After publishing and restarting Vite the problem was eliminated.
